UI/models/Layout.ts

```diff
@@ -1,39 +1,7 @@
-import { JsonAxis, JsonLayout } from "./JsonLayout";
+import * as Plotly from 'plotly.js';
 
-export class Layout {
-    title: string;
-    xaxis: Axis;
-    yaxis: Axis;
+export class Layout implements Partial<Plotly.Layout>{
 
-    constructor(title: string, xaxis: Axis, yaxis: Axis) {
-        this.title = title;
-        this.xaxis = xaxis;
-        this.yaxis = yaxis;
-    }
-
-    public static parseLayout(json: JsonLayout): Layout {
-        const title: string = json.title;
-        const xaxis: Axis = Axis.parseAxis(json.xaxis);
-        const yaxis: Axis = Axis.parseAxis(json.yaxis);
-        return new Layout(title, xaxis, yaxis);
-    }
-}
-
-export class Axis {
-    title: string;
-    showgrid: boolean;
-    zeroline: boolean;
-
-    constructor(titel: string, showgrid: boolean, zeroline: boolean) {
-        this.title = titel;
-        this.showgrid = showgrid;
-        this.zeroline = zeroline;
-    }
-
-    public static parseAxis(json: JsonAxis): Axis {
-        const title: string = json.title;
-        const showgrid: boolean = json.showgrid;
-        const zeroline: boolean = json.zeroline;
-        return new Axis(title, showgrid, zeroline);
+    constructor(title: string) {
```

Comment:
that one looks kind of strange to me, can you explain that a bit? Why do we have a title that is not stored anywhere